substrate/frame/support/src/traits/tokens/fungible/regular.rs

```diff
@@ -260,13 +260,14 @@ where
 	fn burn_from(
 		who: &AccountId,
 		amount: Self::Balance,
+		preservation: Preservation,
```

Comment:
Is the doc comment still correct of this function? I think it should not burn ED anymore when `Preservation` is set to `Protect`.

Reply:
I don't believe comment was valid before either, due to existence of `force`. I've changed it a bit now, hope it's better.